superbench/benchmarks/micro_benchmarks/cuda_memory_bw_performance.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the Cuda memory performance benchmarks."""
+
+import os
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Platform
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class CudaMemBwBenchmark(MicroBenchmarkWithInvoke):
+    """The Cuda memory bus bandwidth performance benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'bandwidthTest'
+        self.__mem_types = ['htod', 'dtoh', 'dtod']
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
```

Comment:
We need to support another mode called 'shmoo':
./bandwidthTest mode=shmoo
Just like the AF mode of RDMA test, it will increase message size and do more tests.

Reply:
sure